.github/workflows/automatic-release.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         uses: "actions/checkout@v2"
 
       - name: "Release"
-        uses: "./"
+        uses: "./.github/actions/automatic-releases/"
```

Comment:
And this part ensures we use that local action, which builds from the Dockerfile directly instead of using the image.